app/page.tsx

```diff
@@ -31,26 +31,41 @@ import {
 } from './filters';
 import { isAddress } from 'viem';
 import Image from 'next/image';
+import { getOpenMarkets } from '@/queries/dune';
+import { Categories } from '@/constants';
 
-const ITEMS_PER_PAGE = 12;
-const SEARCH_DEBOUNCE_DELAY = 600;
+const DEFAULT_CATEGORIES = Object.values(Categories);
+const DEFAULT_CREATOR_OPTION = creatorFilters[0];
 const DEFAULT_ORDER_OPTION = orderFilters[0];
 const DEFAULT_STATE_OPTION = stateFilters[0];
-const DEFAULT_CREATOR_OPTION = creatorFilters[0];
 const DEFAULT_TOKEN_OPTION = tokenFilters[0];
-
-enum Categories {
-  TECHNOLOGY = 'technology',
-  CRYPTO = 'crypto',
-  BUSINESS = 'business',
-  POLITICS = 'politics',
-  ECONOMY = 'economy',
-  INTERNATIONAL = 'international',
-  SPORTS = 'sports',
-  ENTERTAINMENT = 'entertainment',
-}
+const ITEMS_PER_PAGE = 12;
+const SEARCH_DEBOUNCE_DELAY = 600;
 
 type CategoryOptions = Categories | '';
+type DuneCategory = { Category: string };
+
+/**
+ * @param {DunateCategory[]} data - Response from Dune API query
+ * @returns {{ [key in Categories]: number }[]} A sorted array of objects with the count of markets per category
+ */
```

Comment:
I would remove the comment, as it does not add much that we can't see on the function bellow.